### DISCUSSION TITLE
Migrate bridged account to real account

### DISCUSSION THREAD

**Discussion by nichu42** [snarfed/bridgy-fed#1552] [Q&A]
Hi there,

I need to migrate a bridged account to a real BlueSky account (I have a custom domain set up).
Is this possible at all? Is there a tutorial somewhere?

**Comment by snarfed** [comment]
Hi! [This isn't officially supported](https://fed.brid.gy/docs#migration), but if you're technical, you can try this: https://whtwnd.com/bnewbold.net/3l5ii332pf32u#:~:text=Manual%20Account%20Migration . You'll need my help near the end. Let me know if you want to try!

**Reply by nichu42** [reply]
Thanks for the swift response. I definitely wsnt to try! Which variant should I look at - automatic or manual?

**Reply by snarfed** [reply]
Great! Manual, the one I linked to.

**Reply by Loapu** [reply]
Hey, I am trying to migrate, but I have some problems. Would you be available to help me troubleshoot? Maybe Wednesday or Thursday evening (around 6 p.m. CET)?

**Reply by snarfed** [reply]
Sady, after talking with the Bluesky team about this, it turns out that they're not ready for this kind of migration yet after all. Sorry. Hopefully eventually!

**Reply by nichu42** [reply]
Okay, I guess we'll have to start over then. How can we undo the custom domain setting for our bridged account?

**Reply by snarfed** [reply]
Sorry! Just point your DNS or `.well-known/atproto-did` to your new DID, no need to do anything on the Bridgy Fed end.

**Reply by Tamschi** [reply]
Also, don't deactivate the bridge account before you have the handle on the new account.
Apparently Bluesky has some issues where it doesn't release handles of at least deleted accounts correctly.

**Reply by nichu42** [reply]
Thanks for the warning! I am currently waiting for BSky to recognize that the DNS entry has changed...
